### ISSUE TITLE
Too small size for drag and drop area

### ISSUE THREAD

**Issue by petermakowski** [canonical/manager.assets.ubuntu.com#139]
The drop area for file drag-and-drop is too small which results in bad accessibility.

It should be increased according to a11y best practices (or even extended to entire page).

<img width="639" alt="Screenshot 2022-11-24 at 13 36 52" src="https://user-images.githubusercontent.com/7452681/203786316-7130aa56-f9d4-4db3-8249-e74490fe92ce.png">
